src/App.js

```diff
@@ -1,8 +1,19 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { Menu } from "./components/menu/menu";
+import About from "./pages/About";
 
 function App() {
   return (
-    <Menu />
+    <div>
```

Comment:
O ideal não seria o Menu ficar numa componente de ordem maior para n ter q ficar declarando ele em toda page nova que for criar?

Reply:
Certo! Já removi aqui e podemos fazer o mesmo com o rodapé certo?